pnpm-lock.yaml

```diff
@@ -107,7 +107,7 @@ importers:
       '@coremedia/ckeditor5-coremedia-link': link:../packages/ckeditor5-coremedia-link
       '@coremedia/ckeditor5-coremedia-richtext': link:../packages/ckeditor5-coremedia-richtext
       '@coremedia/ckeditor5-coremedia-studio-integration-mock': link:../packages/ckeditor5-coremedia-studio-integration-mock
-      '@coremedia/ckeditor5-data-normalization': link:../packages/ckeditor5-data-normalization
```

Comment:
I stumbled upon this as I would have had expected that there is no reference to this package anymore. 
There are still some dependencies (e.g. https://github.com/CoreMedia/ckeditor-plugins/blob/fix/remove-data-normalization/app/package.json#L45)
Those must be removed as it might cause build-issues later on.